src/renderers/PartialViewRenderers/frontPage/stateListRenderer.js

```diff
@@ -12,7 +12,11 @@ function getStateList() {
     ipcRenderer.once('State:getStateListOK', function (e, res) {
         $('#stateContainer').empty();
         $('#stateContainer').append('<ul id="ulStateList"></ul>');
-
+        res.sort(function(a, b) {
+            var textA = a.stateName.toUpperCase();
+            var textB = b.stateName.toUpperCase();
```

Comment:
Change variable declaration from using **_var_** to using **_let_**. The variable will leak its scoping from local to global.